release/ray_release/scripts/build_pipeline.py

```diff
@@ -46,25 +56,46 @@ def main(test_collection_file: Optional[str] = None):
         # If the Ray test repo is set, we clone that repo to fetch
         # the test configuration file. Otherwise, we might be missing newly
         # added test.
-        tmpdir = tempfile.mktemp()
-
-        clone_cmd = f"git clone --depth 1 --branch {branch} {repo} {tmpdir}"
-        logger.info(f"Cloning test repository: {clone_cmd}")
-        try:
-            subprocess.check_output(clone_cmd, shell=True)
-        except Exception as e:
-            raise ReleaseTestCLIError(
-                f"Could not clone test repository " f"{repo} (branch {branch}): {e}"
-            ) from e
-        test_collection_file = os.path.join(tmpdir, "release", "release_tests.yaml")
+        repo = settings["ray_test_repo"]
```

Comment:
Duplicated

```suggestion
```

Also should this just be `if repo and not no_clone_repo:` in line 45?